lib/geocoder/lookups/here.rb

```diff
@@ -19,48 +19,40 @@ def supported_protocols
     private # ---------------------------------------------------------------
 
     def base_query_url(query)
-      "#{protocol}://#{if query.reverse_geocode? then 'reverse.' end}geocoder.ls.hereapi.com/6.2/#{if query.reverse_geocode? then 'reverse' end}geocode.json?"
+      service = query.reverse_geocode? ? "revgeocode" : "geocode"
+
+      "#{protocol}://#{service}.search.hereapi.com/v1/#{service}?"
     end
 
     def results(query)
       return [] unless doc = fetch_data(query)
-      return [] unless doc['Response'] && doc['Response']['View']
-      if r=doc['Response']['View']
-        return [] if r.nil? || !r.is_a?(Array) || r.empty?
-        return r.first['Result']
-      end
-      []
+      return [] unless doc["items"] && !doc["items"].empty?
```

Comment:
`return x unless A && B` is always a bit hard to read (I even think there might be a rubocop for this), perhaps we could rewrite it with `return [] if ...` ?